Validation/RecoEgamma/python/ElectronMcFakeValidator_gedGsfElectrons_cfi.py

```diff
@@ -57,7 +57,18 @@
   histosCfg = cms.PSet(electronMcFakeHistosCfg)
 )
 
-
-
-
-
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
+phase2_hgcal.toModify(
+    electronMcFakeValidator,
```

Comment:
@beaudett 
here we switch from `gedGsfElectrons` to `ecalDrivenGsfElectronsFromMultiCl` et all for all related collection.
What I was proposing was to clone the original analyzer and run them both at the same time so that `gedGsfElectrons` are left as they are (broken for the time being) and `FromMultiCl` added. The moment we fix the `gedGsfElectrons` there's nothing else/more to touch here. Would that make sense?

Reply:
I see. It would be indeed convenient. But we will have duplicate set of histograms then, one broken (with empty histograms) and one filled. Aren't the broken ones take disk-space for nothing ? If you tell me that it is not an issue, then I buy your solution !

Reply:
@beaudett they will take up both memory and disk space: the latter is very limited, the formes is somehow quite expensive these days. Just to have a rough understanding, of how much histograms are we talking about? I'd then put the histograms in 2 separate folders. That could also be useful to have a comparison later on between the 2 collections.

Reply:
Hi Marco,

we are speaking of > 1K histograms, with a good fraction of them being 2D.
ElectronMcFakeValidator : 360 histos
ElectronMcSignalValidator : 494 histos
ElectronMcSignalValidatorPt1000 : 494 histos
ElectronMcSignalValidatorMiniAOD : 41 histos

Arnaud reminded me that we had been asked in the past to reduce the number of histograms, and we had to struggle a bit when we introduced the Pt1000 duplicated directory (which is not empty) and for which the ranges of the histograms are adapted.
Therefore, and after some thinking, we are a bit reluctant to clone the sequence and to put a new directory in place. We are aware that it implies that we manually switch back to the regular collection once it is in place.
We hope it is agreeable to you.

Florian & Arnaud